kong/conf_loader/init.lua

```diff
@@ -486,6 +486,14 @@ local function load(path, custom_conf, opts)
   -- The socket path is where we store listening unix sockets for IPC and private APIs.
   -- It is derived from the prefix and is NOT intended to be user-configurable
   conf.socket_path = pl_path.join(conf.prefix, constants.SOCKET_DIRECTORY)
+  conf.worker_events_sock = constants.SOCKETS.WORKER_EVENTS
```

Comment:
Note to future us: we should consider some means of passing constants through to the nginx template env so as to not need this kind of code.

Reply:
Agreed!